app/pages/learn/01_tutorial/07_rich_client_apps/02_introduction-javafx-animation.md

```diff
@@ -0,0 +1,265 @@
+In contrast, `AnimationTimer` is designed for frame-by-frame updates and does not make use of `WritableValue<T>`.
+
+## WritableValue
+`WritableValue<T>` is an interface that wraps a value that can be read and set. It is commonly used for storing properties in JavaFX UI elements, like
+`width` or `height` in the [Rectangle](javafxdoc:Rectangle) shape.
+
+## Animation
+
+The [Animation](javafxdoc:Animation) abstract class provides the core functionality for `Transition` and `Timeline` animations.
+
+An `Animation` consists of multiple properties:
+- The `targetFramerate` is the maximum framerate (frames per second) at which this `Animation` will run.
+- The `currentTime` is the current point in time in the `Animation` as a [Duration](javafx:Duration).
+- The `rate` defines the direction and speed at which the `Animation` is expected to be played. It supports both positive and negative numbers.
+- The `cycleCount` defines the number of cycles of this `Animation`. It can't be changed while running and must be positive.
+- The `cycleDuration` is the `Duration` of one cycle of this `Animation`. It is the time it takes to play from start to end of the `Animation` **at the default rate of 1.0**.
+- The `totalDuration` indicates the total duration of this `Animation`, including repeats. It is the result of `cycleDuration * cycleCount` or possibly `Duration.INDEFINITE`.
+- The `delay` is the `Duration` that delays the `Animation` when starting.
+- The `autoReverse` property specifies whether the `Animation` will play in reverse direction on alternating cycles.
+- The `onFinished` event handler is used to define additional behavior when the `Animation` finished.
+- The `status` represents the current state of the `Animation`, possible states are `PAUSED`, `RUNNING` and `STOPPED`.
+
+Additionally, it provides several useful methods, like `play()`, `playFrom(String cuePoint)`, `pause()`, `stop()` and more to control the animations flow.
+A quick look into the JavaDocs provides a great overview of its functionalities.
+
+## Transition
+The [Transition](javafxdoc:Transition) abstract class serves as the foundational class for all transitions, presenting a common form of `Animation`.
+JavaFX provides a variety of built-in transitions for common [Node](javafxdoc:Node) and [Shape](javafxdoc:Shape) properties.
+
+### Fade Transition
+The [FadeTransition](javafxdoc:FadeTransition) creates a fade effect. 
+This is done by updating the `opacity` property of the `Node` at regular intervals.
+
+![FadeTransition](/assets/images/javafx/animation/transition/fade-example.gif)
+```java
+Circle circle = new Circle(150, 150, 20, Color.GREEN);
+
+FadeTransition transition = new FadeTransition(Duration.seconds(5), circle);
+transition.setFromValue(1.0);
+transition.setToValue(0);
+transition.setInterpolator(Interpolator.LINEAR);
+
+transition.play();
+```
```

Comment:
These code snippets are not fully self-contained. That is ok if you link to another article that explains how to get a node onto the screen. Can you look into the articles under https://dev.java/learn/javafx/ and possibly a section therein to link to? (Clicking on a header in the table of contents adds its anchor to the URL.) It's enough to add a little sentence (maybe in parentheses under the snippet) with the link in this section only - no need to repeat everywhere.